docs/src/pages/guides/migration-v4/migration-v4-de.md

```diff
@@ -1898,7 +1898,7 @@ The `@material-ui/styles` package is no longer part of `@material-ui/core/styles
 ```ts
 import { Theme } from '@material-ui/core/styles';
 
-declare module '@material-ui/styles' {
+declare module '@material-ui/styles/defaultTheme' {
```

Comment:
Please revert the changes on the translated files

x the others

Reply:
Should I remove this change from all translated files And just leave this file here?  docs/src/pages/guides/migration-v4/migration-v4.md